packages/arb-token-bridge-ui/src/token-bridge-sdk/BridgeTransferStarterFactory.ts

```diff
@@ -1,29 +1,44 @@
 import {
   BridgeTransferStarter,
-  BridgeTransferStarterProps
+  BridgeTransferStarterProps,
+  BridgeTransferStarterPropsWithChainIds
 } from './BridgeTransferStarter'
 import { EthDepositStarter } from './EthDepositStarter'
 import { Erc20DepositStarter } from './Erc20DepositStarter'
 import { EthWithdrawalStarter } from './EthWithdrawalStarter'
 import { Erc20WithdrawalStarter } from './Erc20WithdrawalStarter'
-import { getBridgeTransferProperties } from './utils'
+import {
+  getBridgeTransferProperties,
+  getProvider,
+  isBridgeTransferStarterPropsWithChainIds
+} from './utils'
+import { Provider } from '@ethersproject/providers'
 
 export class BridgeTransferStarterFactory {
   public static async create(
-    initProps: BridgeTransferStarterProps
+    props: BridgeTransferStarterProps | BridgeTransferStarterPropsWithChainIds
```

Comment:
We have 2 options to initialise the bridge-transfer-factory.
- One with chainIds
- Another one with providers